scripts/json2db.pl

```diff
@@ -0,0 +1,238 @@
+#! /usr/bin/perl -w
+# vim:sw=8:ts=8:et:nowrap
+use strict;
+use utf8;
+
+use FindBin;
+chdir $FindBin::Bin;
+use lib "$FindBin::Bin";
+use lib "$FindBin::Bin/../commonlib/perllib";
+
+use mySociety::Config;
+mySociety::Config::set_file("$FindBin::Bin/../conf/general");
+
+my $parldata = mySociety::Config::get('RAWDATA');
+my $lastupdatedir = mySociety::Config::get('INCLUDESPATH') . "../../../xml2db/";
+
+use DBI;
+use HTML::Entities;
+use File::Find;
+use Getopt::Long;
+use Data::Dumper;
+use File::Slurp::Unicode;
+
+use Uncapitalise;
+use JSON::XS;
+
+use vars qw($motion_count $policy_count $vote_count %motions_seen);
+my $json = JSON::XS->new->latin1;
+
+my $dsn = 'DBI:mysql:database=' . mySociety::Config::get('TWFY_DB_NAME'). ':host=' . mySociety::Config::get('TWFY_DB_HOST');
+my $dbh = DBI->connect($dsn, mySociety::Config::get('TWFY_DB_USER'), mySociety::Config::get('TWFY_DB_PASS'), { RaiseError => 1, PrintError => 0 });
+
+my $policycheck = $dbh->prepare("SELECT policy_id from policies where policy_id = ?");
+my $policyadd = $dbh->prepare("INSERT INTO policies (policy_id, title, description) VALUES (?, ?, ?)");
+my $policyupdate = $dbh->prepare("UPDATE policies SET title = ?, description = ? WHERE policy_id = ?");
+
+my $motioncheck = $dbh->prepare("SELECT division_title, gid, direction, yes_text, no_text FROM policydivisions WHERE division_id = ? AND policy_id = ?");
+
+my $motionadd = $dbh->prepare("INSERT INTO policydivisions (division_id, policy_id, house, direction, division_title, yes_text, no_text, division_date, division_number, gid) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?)");
+my $motionupdate = $dbh->prepare("UPDATE policydivisions SET gid = ?, division_title = ?, yes_text = ?, no_text = ?, direction = ? WHERE division_id = ? AND policy_id = ?");
+
+my $votecheck = $dbh->prepare("SELECT member_id, vote FROM memberdivisionvotes WHERE division_id = ?");
+my $voteadd = $dbh->prepare("INSERT INTO memberdivisionvotes (member_id, division_id, vote) VALUES (?, ?, ?)");
+my $voteupdate= $dbh->prepare("UPDATE memberdivisionvotes SET vote = ? WHERE member_id = ? AND division_id = ?");
+
+my $motionsdir = $parldata . "scrapedjson/policy-motions/";
+
+$motion_count = $policy_count = $vote_count = 0;
+
+foreach my $dreamid (
```

Comment:
If this is the same list as in mpinfoin.pl, could we share it somewhere? Perhaps then even in a CSV file that classes/Policies.php could read in too… (but just sharing between the two Perl files would be good :) )